guides/Testing Community Of Practice

```diff
@@ -0,0 +1,17 @@
+---
+category: Testing Community Of Practice
```

Comment:
the category needs to be `Communities of Practice` rather than `Testing Community Of Practice`